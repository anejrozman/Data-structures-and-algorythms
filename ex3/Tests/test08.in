17,27,2
6.550058225915942,0.6442873192440728,2.9835019973511754,5.41042407808425,8.833572123322316,3.696097132083343,8.384201153669254,7.968702809990957,2.8318540014366587,2.8248452832706596,8.468011223116939,5.089023207428465,6.682808719513508,6.231800580928653,0.6123609132501111,7.930099931320428,2.909734562699643
9,0,514.687
14,8,771.297
7,6,771.983
9,8,387.589
16,8,264.871
16,15,777.946
13,1,268.848
3,12,909.026
10,4,270.16
9,5,17.036
11,13,666.171
12,5,670.554
8,4,293.128
4,0,807.314
2,16,552.957
11,10,429.009
3,5,46.218
4,12,818.376
9,14,829.1
10,16,79.513
11,4,219.179
9,4,96.43
1,8,100.532
13,14,875.126
0,1,630.916
9,10,891.031
1,11,382.323
