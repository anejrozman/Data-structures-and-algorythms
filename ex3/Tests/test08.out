15,2349.3293
2,66.8112
